File: CardGameTracker.Core/Models/Game.cs
namespace CardGameTracker.Models;

public class Game(List<Player> players, User user)
{
    public Guid UserId { get; set; } = user.Id;
    public Guid Id { get; set; } = Guid.NewGuid();
    public List<Player> Players { get; set; } = players;
    

}

File: card-game-tracker-react/src/models/game.ts
import { Player } from "./player";

type GameType = 'Wizard' | 'Hearts';

interface Game{
    UserId: string;
    Id: string;
    Players: Player[];
    CreatedDate: Date;
    LastModifiedDate?: Date;
    GameType: GameType;
}

export type { Game, GameType };